.github/workflows/pre-commit.yaml

```diff
@@ -20,8 +20,9 @@ jobs:
     outputs:
       directories: ${{ steps.dirs.outputs.directories }}
     steps:
+      - run: aws --version
```

Comment:
this jives with the findings - looks like GitHub updated the runners to use the latest awscli 
```sh
aws-cli/2.7.7 Python/3.9.11 Linux/5.13.0-1029-azure exe/x86_64.ubuntu.20 prompt/off
```

Ref: https://github.com/aws-ia/terraform-aws-eks-blueprints/runs/6942642586?check_suite_focus=true